### DISCUSSION TITLE
Reblog with tags

### DISCUSSION THREAD

**Discussion by laughingacademy** [AprilSylph/XKit-Rewritten#145] [Suggestion Box]
Would it be possible to add the option of reblogging a post with the previous poster's tags?

**Comment by AprilSylph** [comment]
I don't believe in any kind of auto-tagging.

**Comment by NyxSloth** [comment]
What about in the context of being able to reblog my own posts with tags I've used in the past? I miss this function dearly :(
Or at least a way to easily click to add the suggestions, currently it creates a long list of suggestions that doesn't really make it easier to add previous tags.

**Reply by marcustyphoon** [reply]
I do think it would be an improvement to roll our own tag suggestion UI. The native one kind of sucks, imo. Not a very easy thing, though.
